test-driver/src/Compare.ts

```diff
@@ -150,8 +150,8 @@ export class Compare {
             let expectedContainer: any = [];
             if (actual.isEmbedded(actualEvent) && expected.isEmbedded(expectedEvent)) {
                 //we found a list of strings that we need to interpret as top level ion text streams.
-                actualContainer = this.parseEmbeddedStream(actualEvent);
-                expectedContainer = this.parseEmbeddedStream(expectedEvent);
+                actualContainer = this.parseEmbeddedStream(actualEvent)
+                expectedContainer = this.parseEmbeddedStream(expectedEvent)
```

Comment:
I'm surprised that `tslint` didn't complain about the missing semicolons and that `prettier` didn't automatically put them back.